README.md

```diff
@@ -213,6 +231,28 @@ batch.get(
 [daa, trx_volume] = batch.execute()
 ```
 
+```python
+from san import AsyncBatch
+
+batch = AsyncBatch()
+
+batch.get(
+    "daily_active_addresses/santiment",
+    from_date="2018-06-01",
+    to_date="2018-06-05",
+    interval="1d"
+)
+
+batch.get(
+    "transaction_volume/santiment",
+    from_date="2018-06-01",
+    to_date="2018-06-05",
+    interval="1d"
+)
+
+[daa, trx_volume] = batch.execute(max_workers=10)
```

Comment:
```suggestion
from san import AsyncBatch

def main():
    batch = AsyncBatch()

    batch.get(
        "daily_active_addresses/santiment",
        from_date="2018-06-01",
        to_date="2018-06-05",
        interval="1d"
    )
    batch.get(
        "transaction_volume/santiment",
        from_date="2018-06-01",
        to_date="2018-06-05",
        interval="1d"
    )
    [daa, trx_volume] = batch.execute(max_workers=10)

    return [daa, trx_volume]

if __name__ == '__main__':
    main()
```

Reply:
Already updated, good finding. I've also added some explanation about that.

Reply:
Actually, after changing from `ProcessPoolExecutor` to `ThreadPoolExecutor` it seems that this is no longer needed.